### PR TITLE
use regular expression to check docker compose or not.

### PR DESCRIPTION
**Pull request by odg0318**
I changed to use regular expression for docker-compose. related to #15

### CODE REVIEW

File: pkg/cmdbuilder/cmdbuilder.go [hunk 1]
```diff
@@ -30,28 +31,37 @@ func (c *Config) Add(t string) {
...
-			compose := false
-			if strings.Contains(c.Params["filters"][0], "[") {
-				compose = true
+			if len(c.Params["filters"]) == 0 {
+				return
+			}
+
+			filter := []byte(c.Params["filters"][0])
+
+			compose, err := regexp.Match(`^\{"(.+)": \["(.+)"]}$`, filter)
```
Comment (juliengk) [inline pkg/cmdbuilder/cmdbuilder.go]: why do you escape opening brackets but not the closing ones?

File: pkg/cmdbuilder/cmdbuilder.go [hunk 2]
```diff
@@ -30,28 +31,37 @@ func (c *Config) Add(t string) {
...
 				var v map[string][]string
 
-				err := json.Unmarshal([]byte(c.Params["filters"][0]), &v)
+				err := json.Unmarshal(filter, &v)
 				if err != nil {
 					panic(err)
 				}
 
 				for k, val := range v {
+					fmt.Println("k, val", k, val)
```
Comment (juliengk) [inline pkg/cmdbuilder/cmdbuilder.go]: can you remove print functions please

File: pkg/cmdbuilder/cmdbuilder.go (hunk 1 above)
Comment (juliengk) [reply pkg/cmdbuilder/cmdbuilder.go]: regexp.Match(`^\{".+":\s*\[.+\]\}$`, filter)

**Comment by odg0318** [comment]
Updated.

**Comment by juliengk** [comment]
Good. Can you squash your commit please.

**Comment by odg0318** [comment]
Squashed.

**Comment by juliengk** [comment]
thanks

**Pull request merged**